OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -390,6 +457,8 @@ void TracerThread::Run(
     perf_event_open_errors |= uprobes_event_open_errors;
   }
 
+  perf_event_open_errors |= !OpenTracepoints(cpuset_cpus);
```

Comment:
Nit: This was already here, but I'm wondering if you could make this more readable by changing this to be:

bool perf_event_open_all_succeeded = true;

Then you don't need the ! and can just write

perf_event_open_all_succeeded &= OpenTracepoints(cpuset_cpus);

Reply:
Yes, I think if we change one, all should be changed. But you make a good point that something uprobes_event_open_succeeded doesn't make sense if you don't actually aim to open uprobes. Let's keep it as it is.